package.json

```diff
@@ -3249,7 +3271,29 @@
           "command": "razor.reportIssue",
           "when": "resourceLangId == aspnetcorerazor"
         }
+      ],
+      "editor/context": [
+        {
+          "command": "dotnet.test.runTestsInContext",
+          "when": "editorLangId == csharp",
+          "group": "2_dotnet@1"
```

Comment:
Vscode groups are documented here: https://code.visualstudio.com/api/references/contribution-points#Sorting-of-groups

`2_` puts them after find all refs/rename, but before copy/paste. The `@` symbol on the end denotes relative ordering of commands in a group.

Reply:
I dig it